scripts/update-policy.py

```diff
@@ -0,0 +1,72 @@
+import requests
+import oyaml
+from itertools import chain
+from collections.abc import Mapping
+
+
+extra_syscalls = [
+  "_exit",
+  "clear_tid_address",
+  "futimens",
+  "getreuid",
+  "inotify1_init",
+  "signal",
+  "sigpending",
+  "sigsuspend",
+  "wait",
+  "wait3"
+]
+
+
+class OYAMLDumper(oyaml.Dumper):
+    def increase_indent(self, flow=False, indentless=False):
+        return super(OYAMLDumper, self).increase_indent(flow, False)
+
+
+def lense(obj, *args):
```

Comment:
I wrote this utility function to simplify codes,
but the below utility function might be better like [dot-prop](https://github.com/sindresorhus/dot-prop).

```
getProperty({foo: {bar: 'unicorn'}}, 'foo.bar');
//=> 'unicorn'

getProperty({foo: {bar: 'a'}}, 'foo.notDefined.deep');
//=> undefined
```

Reply:
We actually already have one implementation :wink:: https://github.com/lablup/backend.ai/blob/e2e18a2/src/ai/backend/common/utils.py#L107

Reply:
If there is a well-packaged 3rd-party library with a clean API, I think it is also good to use one.

Reply:
I copied and pasted the `nmget` code, but maybe making it as a separate lib name `nmget` would be better idea to maintain.